docs/en/observability/monitor-uptime-synthetics.asciidoc

```diff
@@ -1,50 +1,92 @@
+By adding the Elastic Synthetics integration to the policy used by your agent, you can monitor the
+availability of network endpoints and services using the following Uptime monitors:
 
-The {uptime-app} in {kib} includes two types of synthetic monitoring:
-
-* <<monitoring-synthetics>>
 * <<monitoring-uptime>>
+* <<monitoring-synthetics>>
+
+[discrete]
+[[monitoring-uptime]]
+== Lightweight HTTP/S, TCP, and ICMP monitors
+
+In the {uptime-app}, you can monitor the status of network endpoints using the following lightweight checks:
+
+|===
+
+| *ICMP monitor* | Check the availability of your hosts. The ICMP monitor uses ICMP (v4 and v6) Echo
+Requests to check the network reachability of the hosts you are pinging. This will tell you whether the
+host is available and connected to the network. However, it doesn't tell you if a service on the host is running or
+not.
+
+| *TCP monitor* | Monitor the services running on your hosts. The TCP monitor checks individual ports
+to ensure the service is accessible and running.
+
+| *HTTP monitor* | Monitor your website. The HTTP monitor checks to ensure specific endpoints return the correct
+status code and display the correct text.
+
+|===
+
+[role="screenshot"]
+image::images/uptime-app.png[Uptime app in Kibana]
+
+To set up your first monitor, see <<uptime-set-up>>.
+
+// Along with getting notified when your <<tls-certificate-alert,TLS certificates>> are set to expire, you can also
+// create an <<duration-anomaly-alert,Uptime duration anomaly >> rule to receive notifications based on the response durations for all of the
+// geographic locations of each monitor.
 
 [discrete]
 [[monitoring-synthetics]]
-== Browser checks
+== Browser monitors
 
 beta[] Real browser synthetic monitoring enables you to test critical actions and requests that an end-user would perform
-on your site at predefined intervals and in a controlled environment. You can view each synthetic monitoring journey
-in the {uptime-app} side-by-side with your other <<monitor-uptime,Uptime monitors>>. The result is rich, consistent, and repeatable
-data that you can trend and alert on.
+on your site at predefined intervals and in a controlled environment.
+The result is rich, consistent, and repeatable data that you can trend and alert on.
 
-For example, test popular user journeys, like logging in, adding items to a cart, and checking
+For example, you can test popular user journeys, like logging in, adding items to a cart, and checking
 out -- actions that need to work for your users consistently.
 
-Alerting ensures that any degraded performance or broken actions are fixed before impacting your bottom line or customer
-experience.
+Using the synthetics agent, you can run automated synthetic monitoring test suites on a real Chromium browser and
+view each synthetic monitoring journey in the {uptime-app} side-by-side with your other monitors.
```

Comment:
This might be a little confusing to users because what actually gets journeys to the Uptime app is Heartbeat (or the Elastic Agent, which bundles Heartbeat).

I'd probably say that the same journeys people write and run directly with the agent could also be run using Heartbeat, so that people see results in the uptime app (at the end of the day Heartbeat also uses the agent, but that's unnecessary info, so we don't need to say that here)

Reply:
I made an attempt, but I'm not sure if it addresses your comment.